demo/alloy-showcase-webapp/src/main/webapp/views/component.xhtml

```diff
@@ -43,18 +45,17 @@
 			</alloy:tab>
 		</alloy:tabView>
 		<alloy:buttonRow styleClass="text-center">
-			<c:set var="showcaseComponentSuite" value="alloy" />
-			<c:set var="showcaseSourceContainer" value="webapp" />
-			<c:if test="#{selectedComponentPrefix == 'bridge' || selectedComponentPrefix == 'portal' || selectedComponentPrefix == 'portlet'}">
-				<c:set var="showcaseComponentSuite" value="portal" />
-				<c:set var="showcaseSourceContainer" value="portlet" />
+			<c:set var="sourceControlURL"
+				value="https://github.com/liferay/liferay-faces-alloy/edit/master/demo/alloy-showcase-webapp/src/main/webapp/WEB-INF/component/alloy/#{selectedComponentLowerCaseName}/#{selectedComponentUseCaseName}/#{selectedComponentCamelCaseName}.xhtml" />
```

Comment:
@stiemannkj1 Does the URL need to be different in the 2.x branch? If so, please send a PR. Thanks.